File: TaskBoard.AndroidApp/app/src/main/java/taskboard/androidclient/data/Task.java
package taskboard.androidclient.data;

public class Task {
    private int id;
    private String title;
    private String description;
    private String createdOn;
    private String board;
    private User owner;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getCreatedOn() {
        return createdOn;
    }

    public String getBoard() {
        return board;
    }

    public void setBoard(String board) {
        this.board = board;
    }

    public User getOwner() {
        return owner;
    }
}

File: TaskBoard.DesktopApp/Data/Task.cs
namespace TaskBoard.DesktopApp.Data
{
    public class Task
    {
        public int Id { get; init; }
        public string Title { get; init; }
        public string Description { get; init; }
        public string CreatedOn { get; init; }
        public string Board { get; init; }
        public User Owner { get; init; }
    }
}